### ISSUE TITLE
Make --irods-package-version optional with --use-static-image by providing a strong default of "latest release"

### ISSUE THREAD

**Issue by alanking** [irods/irods_testing_environment#121]

**Comment by alanking** [comment]
This may require changes to how `--irods-package-version` works. The package version for iRODS packages include information such as the build number and the platform (in some cases). The end user (i.e. myself) would probably prefer to just enter the release number and have the system work out the rest (e.g. 4.3.0 vs 4.3.0-1~focal).

Because of this, the build arguments for the static images as well as the iRODS package version argument. It will be difficult to work out which build number to use if we ever release more than 1 per platform.

**Comment by trel** [comment]
> It will be difficult to work out which build number  

can it somehow ping/gather the 'latest' from the truth of packages.irods.org?

**Comment by alanking** [comment]
I suppose we could, yes.

Would it be worth providing the ability for the user to specify a build number other than the latest one?

If not, then the difficulty is avoided because we can just gather the info from packages.irods.org or even hard-code the info in the environment.

If so, having a system that can both derive and allow one to specify may prove difficult with respect to the static images since conditionals are not a thing in Dockerfiles. Would require some rethinking on the current approach.

**Comment by trel** [comment]
yeah, we could ignore the build number requested for the static ones - you get the latest.   if you're looking to build your own images... sure, we'll listen.
